assets/prefabs/Handwhisk.prefab

```diff
@@ -0,0 +1,26 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking:handwhisk",
```

Comment:
Works well in-game, but we need a quick attribution line in the README even if it is self-created so we can credit the work :-)